scripts/scripts.js

```diff
@@ -1359,8 +1374,10 @@ sampleRUM.drain('convert', (cevent, cvalueThunk, element, listenTo = []) => {
           sampleRUM('variant', { source: experiment, target: treatment });
         });
       // send conversion event
-      const cvalue = typeof cvalueThunk === 'function' ? cvalueThunk() : cvalueThunk;
-      sampleRUM('convert', { source: cevent, target: cvalue, element: celement });      
+      const cvalue = typeof cvalueThunk === 'function' ? cvalueThunk(element) : cvalueThunk;
+      if (cvalue || cevent) {
```

Comment:
I don't think the `if` is necessary, we can discard unlabeled conversions in the analytics report.

Reply:
While testing, a `convert` ping was sent without source or target, that looked weird to my eyes. It can easily happen if the author misconfigures the metadata property. This is why I added the `if`

Reply:
Removing the `if` as agreed on slack